pkg/acceptance/bettertestspoc/assert/resourceassert/tag_resource_gen.go

```diff
@@ -71,11 +71,6 @@ func (t *TagResourceAssert) HasSchemaString(expected string) *TagResourceAssert
 // Attribute empty checks //
 ////////////////////////////
 
-func (t *TagResourceAssert) HasNoAllowedValues() *TagResourceAssert {
```

Comment:
these two should not be deleted because this is generated, right?

Reply:
Reverted.